cluster/service/cluster_service_blackbox_test.go

```diff
@@ -399,6 +399,43 @@ func (s *ClusterServiceTestSuite) TestCreateOrSaveClusterFromEndpoint() {
 	})
 }
 
+func (s *ClusterServiceTestSuite) TestLoad() {
+
+	s.T().Run("ok", func(t *testing.T) {
+		// given
+		c := newTestCluster()
+		err := s.Application.ClusterService().CreateOrSaveCluster(context.Background(), c)
+		require.NoError(t, err)
+		// when
+		result, err := s.Application.ClusterService().Load(context.Background(), c.ClusterID)
+		// then
+		require.NoError(t, err)
+		require.NotNil(t, result)
+		assert.Equal(t, c.Name, result.Name)
```

Comment:
You can use `test.AssertEqualClusters(t, c, result)` instead.

Reply:
ok, let me do that

Reply:
addressed in 13e4d5c